lib/rpc/server/agent_test.go

```diff
@@ -455,3 +458,7 @@ func (r *peerStore) remove(peer Peer) {
 	defer r.Unlock()
 	delete(r.peers, peer.Addr())
 }
+
+func isPeerDeniedError(err error) bool {
+	return strings.Contains(err.Error(), "peer not authorized")
```

Comment:
Nit: "peer not authorized" should probably be a constant, or the approach where there is a pre-created error that is returned and used for comparison. var ErrPeerNotAuthorized...

Reply:
I removed the comparison with a string literal.